gateway-js/src/__tests__/gateway/reporting.test.ts

```diff
@@ -234,21 +234,49 @@ describe('reporting', () => {
         "tracesPerQuery": Object {
           "# -
       {me{name{first last}}topProducts{name}}": Object {
+            "referencedFieldsByType": Object {
```

Comment:
Pretty sure this is just because we're upgrading the version of `graphql-tag` that we're using. Seemed like the change was benign, so I just let the snapshot update.

Reply:
Possibly a related consequence of. I don't think it's problematic.